homeassistant/components/yolink/strings.json

```diff
@@ -75,5 +75,33 @@
         "name": "Volume"
       }
     }
+  },
+  "services": {
+    "play_on_speaker_hub": {
+      "name": "Play on SpeakerHub",
+      "description": "Convert text to audio play on YoLink SpeakerHub",
+      "fields": {
+        "target_device": {
+          "name": "SpeakerHub Device",
+          "description": "SpeakerHub Device"
+        },
+        "message": {
+          "name": "Text Message",
+          "description": "Text Message"
+        },
+        "tone": {
+          "name": "Tone",
+          "description": "Tone"
+        },
+        "volume": {
+          "name": "Volume",
+          "description": "Volume"
+        },
+        "repeat": {
+          "name": "Repeat",
+          "description": "Repeat Times"
```

Comment:
I think we can be more descriptive in the description. "Repeat times" -> "The amount of times the text will be repeated."